### PR TITLE
Reminder email with direct cancel

### PR DESCRIPTION
**Pull request by icstunna**
* User is now permitted to cancel RSVP's without authentication via link in email
* RSVP now has a cancelation token that is sent through the email
* Created a view that confirms whether the user wishes to cancel their RSVP event

### CODE REVIEW

**Comment by ultrasaurus** [comment]
This is such a great idea!

File: config/routes.rb [hunk 1]
```diff
@@ -31,14 +31,16 @@
...
       new do
         get :volunteer
         get :learn
       end
+
       resources :surveys, only: [:new, :create]
     end
-
-    resources :surveys, only: [:new, :index]
+ resources :surveys, only: [:new, :index]
```
Comment (tjgrathwell) [inline config/routes.rb]: This extra route for `surveys` got mysteriously added? Is this a mistake?

**Comment by tjgrathwell** [comment]
Hi @icstunna, I've left a couple of comments above on your changes.

Additionally, can you remove the line "***This now passes all tests.***" from the commit message? Everything that gets merged into master should pass all tests, there's no need to point it out specifically :smiley:

**Comment by tjgrathwell** [comment]
Committed and deployed with amendments as d4920be56ab985acb5d4a7b1833fcf12ea01011f
:tada: Thanks! :tada:

**Pull request closed without merging**